### PR TITLE
Skymaker 3.10.5 (new formula)

### PR DESCRIPTION
**Pull request by damleborgne**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/skymaker.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Skymaker < Formula
+  desc "Generates fake astronomical images"
+  homepage "https://www.astromatic.net/software/skymaker"
+  url "http://www.astromatic.net/download/skymaker/skymaker-3.10.5.tar.gz"
```
Comment (Bo98) [inline Formula/skymaker.rb]: Can this use HTTPS?

**Review by Bo98 — CHANGES_REQUESTED** [review]

File: Formula/skymaker.rb (hunk 1 above)
Comment (damleborgne) [reply Formula/skymaker.rb]: yes, sorry. I am commiting an updated version of the formula with https

**Review by Bo98 — APPROVED** [review]

**Comment by SMillerDev** [comment]
Can you squash your commits into one? Then we can merge this

**Comment by chenrui333** [comment]
Thanks @damleborgne for your contribution to Homebrew! :tada: 🥇 

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by chenrui333** [comment]
@BrewTestBot test this please

**Pull request closed without merging**